config-xml/addon/src/main/java/org/jboss/windup/config/parser/metadata/MetadataSourcePlatformHandler.java

```diff
@@ -0,0 +1,32 @@
+package org.jboss.windup.config.parser.metadata;
+
+import org.apache.commons.lang.StringUtils;
+import org.jboss.windup.config.exception.ConfigurationException;
+import org.jboss.windup.config.metadata.TechnologyReference;
+import org.jboss.windup.config.parser.ElementHandler;
+import org.jboss.windup.config.parser.NamespaceElementHandler;
+import org.jboss.windup.config.parser.ParserContext;
+import org.w3c.dom.Element;
+
+@NamespaceElementHandler(elementName = "sourcePlatform", namespace = "http://windup.jboss.org/v1/xml")
+public class MetadataSourcePlatformHandler implements ElementHandler<TechnologyReference>
+{
+
+    private static String ID="id";
+    private static String VERSION_RANGE="versionRange";
+    
+    @Override
+    public TechnologyReference processElement(ParserContext handlerManager, Element element) throws ConfigurationException
+    {
+        
+        String id = element.getAttribute(ID);
+        String versionRange = element.getAttribute(VERSION_RANGE);
+        if (!StringUtils.isBlank(id) && !StringUtils.isBlank(id))
```

Comment:
Need to throw a validation exception if this value is missing. (Like our existing handlers.)